<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DSSS_Window</class>
 <widget class="QMainWindow" name="DSSS_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DSSS_Window</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="centralLayout">
    <item row="2" column="0" colspan="2">
     <widget class="QPushButton" name="p_startServer">
      <property name="text">
       <string>Start Server</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QListWidget" name="p_activeNodes">
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QPushButton" name="p_stopServer">
      <property name="text">
       <string>Stop Server</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="p_sendAliveNodeRequest">
      <property name="text">
       <string>Send aliveNode request</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" rowspan="2">
     <widget class="QTextEdit" name="p_Log">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
